Code/GraphMol/Wrap/SubstanceGroup.cpp

```diff
@@ -23,6 +24,16 @@ namespace python = boost::python;
 namespace RDKit {
 
 namespace {
+
+  template< typename T >
+  inline
+  std::vector< T > to_std_vector( const python::object& iterable )
```

Comment:
This is a nicelly cleaned up rewrite of the second overload of `Code/RDBoost/Wrap.h:pythonObjectToVect()`
I think it would make sense to just replace that code with this.
The third overload can probably be done more or less the same way. And maybe there's a way to do the first as well.

Reply:
I have given it a try, but I can only substitute the 3rd overload. Apparently, the second one is used to translate some classes that seem to lack some required infrastructure (e.g. Point2D and something else I can´t remember anymore).

Reply:
Scratch that. I got it working on the second try. I guess I messed up something else?

Let me know if the refactor for the first overload is crazy.